package.json

```diff
@@ -105,5 +107,8 @@
   },
   "engines": {
     "node": ">=16.0.0"
+  },
+  "ava": {
+    "verbose": true
```

Comment:
Like our other repositories, moving the ava configuration to `package.json` solves some issues with pure ES modules.